Week3/fuel.py

```diff
@@ -0,0 +1,22 @@
+
+def main():
+    gauge = fuel_tank()
+    if gauge <= 1:
+        print("E")
+    elif gauge >= 99:
+        print("F")
+    else:
+        print(gauge,"%")
+
+def fuel_tank():
+    while True:
+        fuel = input("Fraction: ")
+        try:
+            numerator, denominator = fuel.split("/")
+            f = int(numerator) / int(denominator)
+            if numerator <= denominator and denominator != 0:
```

Comment:
at this point, `numerator` and `denominator` automatically converted by python?  otherwise I don't think you can compare a string to number 0.